packages/did/src/Did.signature.ts

```diff
@@ -12,78 +12,44 @@ import {
   DidResolve,
   DidResourceUri,
   DidSignature,
-  DidVerificationKey,
   UriFragment,
   VerificationKeyRelationship,
 } from '@kiltprotocol/types'
-import { Crypto } from '@kiltprotocol/utils'
+import { Crypto, SDKErrors } from '@kiltprotocol/utils'
 
 import { resolve } from './DidResolver/index.js'
-import { parseDidUri, isKiltDidUri } from './Did.utils.js'
+import { parseDidUri, validateKiltDidUri } from './Did.utils.js'
 import * as Did from './index.js'
 
-type DidSignatureVerificationFromDetailsInput = {
+export type DidSignatureVerificationInput = {
   message: string | Uint8Array
-  signature: string
-  keyId: DidVerificationKey['id']
+  signature: DidSignature
   expectedVerificationMethod?: VerificationKeyRelationship
-  did: DidDocument
+  didResolve?: DidResolve
 }
 
-export type VerificationResult = {
-  verified: boolean
-  reason?: string
-  did?: DidDocument
-  key?: DidVerificationKey
+// Used solely for retro-compatibility with previously-generated DID signatures.
```

Comment:
Is there the `@deprecated` tag for this purpose?

Reply:
I don't think that's necessary bc we don't export this type